horizon/state.py

```diff
@@ -223,20 +211,21 @@ async def _report(self, state: Optional[PersistentState] = None):
                 raise RuntimeError("Unable to post PDP state update to server.")
 
     async def seen_sdk(self, sdk: str):
-        if not sdk in self._state.seen_sdks:
+        if sdk not in self._state.seen_sdks:
             # ensure_future is expensive, only call it if actually needed
-            asyncio.ensure_future(self._report_seen_sdk(sdk))
+            async with self._write_lock:
+                self._tasks.append(asyncio.create_task(self._report_seen_sdk(sdk)))
```

Comment:
wasn't this later removed or somehting like that ? in favor of simply calling the report_seen_sdk

Reply:
Correct
https://github.com/permitio/PDP/pull/245/commits/b02efbb709dcefb29348818488d50d376d0b27e5